tools/internal-schema/src/proto.rs

```diff
@@ -228,24 +228,34 @@ fn interpolate_tokens(
 
     if let Some(v) = version.as_version() {
         let major = v.major.to_string();
-        let major_minor = format!("{}.{}", v.major, v.minor);
-        let year_month = format!("{:0>4}-{:0>2}", v.major, v.minor);
+        let minor = v.minor.to_string();
+        let patch = v.patch.to_string();
+        let major_minor = format!("{}.{}", v.major, v.minor); // Deprecated, remains for backwards compatibility
+        let year_month = format!("{:0>4}-{:0>2}", v.major, v.minor); // Deprecated, remains for backwards compatibility
```

Comment:
@tom-fletcher The `versionYear`, `versionMonth`, and `versionDay` tokens should probably have the zero padding.

Reply:
I can't disagree 😆 I only left these due to the existing `{versionYear}` but this is a good opportunity to add this.